internal/command/migrate_to_v2/migrate_to_v2.go

```diff
@@ -421,6 +446,29 @@ func (m *v2PlatformMigrator) Migrate(ctx context.Context) (err error) {
 		return abortedErr
 	}
 
+	if m.isPostgres {
+		inRegionDbs := m.inRegionMachines()
+
+		err = m.waitForHealthyPgs(ctx)
+		if err != nil {
+			return err
+		}
+
+		var dbUIDs []string
+		tb.Detail("Collecting info about the new pg cluster")
+		dbUIDs, err = m.getPgDBUids(ctx, inRegionDbs)
+		if err != nil {
+			return err
+		}
+
+		tb.Detailf("We are checking the following DB UIDs %+v", dbUIDs)
+
+		err = m.waitForPGSync(ctx, dbUIDs)
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
do we want this after the app unlock? it seems like we'd want this before that if at all possible, but I'm not entirely qualified to make that judgement :)

Reply:
I want it after the release is done, which currently happens after the lock so I'm just inheriting. If we can move the release up it would be sweet to move this up with it